hydromt_wflow/workflows/soilgrids.py

```diff
@@ -188,7 +188,7 @@ def pore_size_distribution_index_layers(ds, thetas):
     ds: xarray.Dataset
         Dataset containing soil properties at each soil depth [sl1 - sl7].
     thetas: xarray.Dataset
-        Dataset containing thetaS at each soil layer depth.
+        Dataset containing soil_theta_s at each soil layer depth.
```

Comment:
THe hydromt name is a common literature name for this parameter (adding soil_ for wflow order of maps in staticmaps is less common)
```suggestion
        Dataset containing theta_s at each soil layer depth.
```